server/test/api/enterprises.js

```diff
@@ -0,0 +1,110 @@
+/*global describe, it, beforeEach, process*/
+
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var expect = chai.expect;
+chai.use(chaiHttp);
+
+// workaround for low node versions
```

Comment:
You should use test helpers to remove this excessive and redundant code.

Here's how to use test helpers
```js
/**
* The following lines will do all this:
* if (!global.Promise) {
*   var q = require('q');
*   chai.request.addPromises(q.Promise);
* }
* process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;
*/
var helpers = require('./helpers');
helpers.configure(chai);
```